app/assets/v2/js/pages/bounty_details.js

```diff
@@ -259,6 +334,7 @@ var pendingChangesWarning = function(issueURL, last_modified_time_remote, now){
         var is_changing_remote_recent = remote_delta < (60 * 60); // less than one minute
 
         should_display_warning = !last_modified_time_remote || ((is_changing_local_recent) && (remote_delta > local_delta));
+        // should_display_warning = false;  // hack to work around the broken warning banner
```

Comment:
whats broken?  is this the thing we talked about on slack?

might be good to just rewrite this code..

Reply:
That comment should be removed.  I was able to work around the problem.